docs/managing-findings.md

```diff
@@ -0,0 +1,108 @@
+# Managing findings
+
+[TOC]
+
+# Findings
```

Comment:
I think you can make this heading an H2 (`## Findings`) and that would flatten the TOC by one level. I realize the content that follows this heading is all about findings, however so is the whole page, and that already has an H1 (“Managing findings”).

Reply:
Good call, I forgot about "Managing findings" above :+1:

Reply:
Reverted back... looks like moving to `h2` removes the `TOC`? It also looks like all other pages use another `h1` below the TOC too :man_shrugging:

Reply:
I’m sorry, my original suggestion was incorrect. Instead of making that line Findings an H2, the correct change is to make all the section headings H1s (and the subsections become H2s):

![Screen Shot 2021-02-09 at 12 47 33 PM](https://user-images.githubusercontent.com/302941/107426372-fc186400-6ad4-11eb-97e4-1be28589e37b.png)